python3/pkg.yaml

```diff
@@ -14,7 +15,8 @@ steps:
         sha512: "{{ .python_sha512 }}"
     env:
       SOURCE_DATE_EPOCH: {{ .BUILD_ARG_SOURCE_DATE_EPOCH }}
-      LD_LIBRARY_PATH: "/toolchain/lib"
+      # FIXME: usrmerge in toolchain (?)
+      LD_LIBRARY_PATH: /usr/lib64:/usr/lib:/lib64:/lib
```

Comment:
I think we need to fix this to have less of this downstream

Reply:
StageX team members told me to move GCC's outputs around before I package the toolchain. Will fix in toolchain (also musl broke usrmerge there a bit, yet this broke nothing) and come back here